.golangci.yml

```diff
@@ -214,11 +283,11 @@ linters:
     - revive
     - rowserrcheck
     - staticcheck
-    - structcheck
+    # - structcheck
     - stylecheck
     - typecheck
     - unconvert
     - unparam
     - unused
-    - varcheck
+    # - varcheck
```

Comment:
why commenting these linters out?

Reply:
Via slack: https://elastic.slack.com/archives/CFWBYD4SJ/p1687899013040079?thread_ts=1687890348.365499&cid=CFWBYD4SJ

These are deprecated linters

Reply:
```
WARN [runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused.
WARN [runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused.
WARN [runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused.
```